.github/workflows/compile.yml

```diff
@@ -0,0 +1,106 @@
+name: Build binaries
+
+on: [push, pull_request]
+
+jobs:
+
+  build-linux-aarch64:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          rustup target add aarch64-unknown-linux-gnu
+          sudo apt-get update ; sudo apt-get install gcc-aarch64-linux-gnu binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu
+      
+      - name: Compile
+        env:
+          CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: /usr/bin/aarch64-linux-gnu-gcc
+        run: |
+          cargo build --release --target=aarch64-unknown-linux-gnu
+
+      - name: Copy around
+        run: | 
+          mkdir release
+          cp target/aarch64-unknown-linux-gnu/release/hanachan release/hanachan_linux_aarch64 || /bin/true
+
+      - name: Upload distribution
+        uses: actions/upload-artifact@v2
+        with:
+          name: release
+          path: | 
+            release
+
+
+  build-linux-windows:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          rustup target add i686-unknown-linux-gnu
+          rustup target add x86_64-unknown-linux-gnu
+          #
+          rustup target add i686-pc-windows-gnu
+          rustup target add x86_64-pc-windows-gnu
+          sudo apt-get update ; sudo apt-get install mingw-w64 gcc-multilib
+      
+      - name: Compile
+        run: |
+          cargo build --release --target=i686-unknown-linux-gnu
+          cargo build --release --target=x86_64-unknown-linux-gnu
+          #
+          RUSTFLAGS="-C panic=abort -C embed-bitcode=yes -C lto" cargo build --release --target=i686-pc-windows-gnu
+          cargo build --release --target=x86_64-pc-windows-gnu
+
+      - name: Copy around
+        run: | 
+          mkdir release
+          cp target/i686-unknown-linux-gnu/release/hanachan release/hanachan_linux_i686 || /bin/true
+          cp target/x86_64-unknown-linux-gnu/release/hanachan release/hanachan_linux_x86_64 || /bin/true
+          #
+          cp target/x86_64-pc-windows-gnu/release/hanachan.exe release/hanachan_windows64.exe || /bin/true
+          cp target/i686-pc-windows-gnu/release/hanachan.exe release/hanachan_windows32.exe || /bin/true
```

Comment:
Please use the same suffixes as Linux, for consistency and because we might want to build an aarch64 version for Windows at some point.